# return the absolute angular from scorpiobase's position （x , y ）, yaw indicate yaw
float32 x
float32 y
float32 yaw
